Formula/sratoolkit.rb

```diff
@@ -12,7 +12,11 @@ class Sratoolkit < Formula
     sha256 cellar: :any_skip_relocation, mojave:   "b9b36c564d65e79d3726e4de473e18d892f109221f45dbb91768ecac5758df59"
   end
 
-  depends_on "hdf5"
+  # Failed to build with `hdf5` at ncbi-vdb-source/libs/hdf5/hdf5dir.c:295:89:
+  # error: too few arguments to function call, expected 5, have 4
+  # herr_t h5e = H5Oget_info_by_name( self->hdf5_handle, buffer, &obj_info, H5P_DEFAULT );
+  # Try updating to `hdf5` on future release.
+  depends_on "hdf5@1.10"
```

Comment:
Usually there are build flags we can set to get `hdf5` to behave like earlier versions. We might want to do something like that here.

Reply:
Not sure how to exactly pass build flags into the upstream's build system.

For example, on Linux, I gave up on trying to get `hdf5` to link since I couldn't figure out how to pass in `-lsz`, which is necessary since we build `hdf5` with `szip`.

I can give the common environment variable flags a few tries.

Reply:
Just tried `ENV.prepend "CXXFLAGS", "-DH5_USE_110_API"` and it failed with same error. Will check a few other ways.